src/configuration/keybindings.jsonc

```diff
@@ -119,6 +119,17 @@
         "command": "showPrevParameterHint",
         "when": "editorFocus && parameterHintsMultipleSignatures && parameterHintsVisible"
     },
+    // File Explorer navigation with ctrl+h/k
+    {
+        "key": "ctrl+j",
+        "command": "list.focusDown",
+        "when": "explorerViewletVisible && filesExplorerFocus && !explorerResourceIsRoot && !explorerResourceReadonly && !inputFocus"
```

Comment:
Wondering what's the significance of `explorerResourceIsRoot && !explorerResourceReadonly`? Can it be `filesExplorerFocus && !inputFocus`?